web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -142,6 +142,8 @@ object LiftRules extends LiftRulesMocker {
    */
   type StatelessTestPF = PartialFunction[List[String], Boolean]
 
+  type StackTrace = Array[StackTraceElement]
```

Comment:
It adds a layer of indirection for a very simple type, resulting in confusion about what the type is. I'd rather see `Array[StackTraceElement]` than `StackTrace` -> look up what that means -> realize it's just an `Array[StackTraceElement]` any day of the week.

Might be personal preference though. I almost never use type aliases.